components/content/dancing-blob/musicBlob.vue

```diff
@@ -0,0 +1,205 @@
+<script lang="ts" setup>
+import * as THREE from 'three';
```

Comment:
Just to little things here.

Please import only what you need, in this case is just a vector2 from 'three'

This help with the final bundle